src/bp-forums/common/shortcodes.php

```diff
@@ -727,8 +733,14 @@ public function display_search( $attr, $content = '' ) {
 				return $content;
 			}
 
+			global $post;
+			$has_bbp_search_shortcode = false;
+			if ( ! empty( $post->post_content ) && has_shortcode( $post->post_content, 'bbp-search' ) ) {
+				$has_bbp_search_shortcode = true;
+			}
+
 			// Bail if search is disabled.
-			if ( ! bbp_allow_search() ) {
+			if ( ! $has_bbp_search_shortcode && ! bbp_allow_search() && empty( $attr['search'] ) ) {
```

Comment:
Revert the code changes to allow the search for the shortcode.
We have the option to enable/disabled search. https://prnt.sc/toNUTFeVrvjK